src/sentry/static/sentry/app/views/performance/trends/utils.tsx

```diff
@@ -153,15 +155,38 @@ export function transformDeltaSpread(
   );
 }
 
+export function getTrendProjectId(
+  trend: NormalizedTrendsTransaction,
+  projects?: Project[]
+): string | undefined;
+
+export function getTrendProjectId(
+  trend: NormalizedProjectTrend,
+  projects?: Project[]
+): string | undefined;
```

Comment:
This function is defined 3 times.

Reply:
I was playing with overloading, I originally overloaded `normalizeTrends` which was helpful but got a little carried away and overloaded this as well, I'll peel this one off but am going to leave it on `normalizeTrends`